Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertColumn.cs

```diff
@@ -111,7 +111,10 @@ public ApplicationObject ColumnToNative(Column speckleColumn)
         }
         isUpdate = true;
       }
-      catch { }
+      catch (Autodesk.Revit.Exceptions.ApplicationException)
+      {
+        //something went wrong, re-create it
+      }
```

Comment:
same update behaviour, we should double check what happens if this fails unexpectedly